doc/classes/AnimationNodeStateMachine.xml

```diff
@@ -170,7 +170,7 @@
 		<method name="set_graph_offset">
 			<return type="void">
 			</return>
-			<argument index="0" name="name" type="Vector2">
+			<argument index="0" name="offset" type="Vector2">
```

Comment:
Parameter names are taken from the source code, so this needs to be fixed in the AnimationNodeStateMachine `_bind_methods`.